var/spack/repos/builtin/packages/g2c/package.py

```diff
@@ -46,10 +46,17 @@ class G2c(CMakePackage):
         description="Build and install some utility programs",
         when="@develop",
     )
+    variant(
+        "build_g2c",
+        default=False,
+        description="Build new g2c API, experimental until 2.0.0 release",
+        when="@develop",
+    )
 
     depends_on("libpng", when="+png")
     depends_on("jasper", when="+jasper")
     depends_on("openjpeg", when="+openjpeg")
+    depends_on("libxml2@2.9:")
```

Comment:
I think this is just for develop, is that correct? If so then `depends_on("libxml2@2.9:", when="@develop")`

Reply:
Thanks, good catch! I've made this change.